Makefile

```diff
@@ -1,9 +1,15 @@
-ETHEREUM_IBC_PROTO ?= ./ethereum-ibc-rs/proto/definitions
```

Comment:
You also need revert this line.

Reply:
Revert path
https://github.com/datachainlab/ethereum-ibc-relay-prover/pull/3/commits/4361f6b7b6141dbf088cc162572a7b833e66118f